config/routes.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
```

Comment:
- [ ] Your _routes_ are **not properly configured** with the _devise_ gem, and hence your app is showing a blank screen when launched. Kindly run the command `rails routes` in your terminal to determine which routes are working correctly, and fix the _root_ route accordingly

Reply:
Thank you for letting me know. I fixed this issue and now everything is working properly 👍